code/datums/mutantraces.dm

```diff
@@ -493,6 +493,124 @@
+			detail_over_suit.color = fix_colors(aH.customization_first_color)
+
+			// detail_1.color = aH.customization_first_color
+			// detail_2.color = aH.customization_second_color
+			// detail_3.color = aH.customization_third_color
+
+			mob.update_face()
+			mob.update_body()
+			mob.update_clothing()
+
+	proc/fix_colors(var/hex)
+		var/list/L = hex_to_rgb_list(hex)
+		for (var/i in L)
+			L[i] = min(L[i], 250)
+			L[i] = max(L[i], 5)
+		if (L.len == 3)
+			return rgb(L["r"], L["g"], L["b"])
+		return rgb(22, 210, 22)
+
+	sight_modifier()
+		mob.see_in_dark = SEE_DARK_HUMAN + 1
+		mob.see_invisible = 1
+		
+	firevuln = 1.1
+
+	say_filter(var/message)
+		return replacetext(message, "r", stutter("rr"))
+		
+	emote(var/act)
+		var/message = null
+		switch(act)
+			if("scream")
+				if (narrator_mode)
+					if(mob.emote_allowed)
+						mob.emote_allowed = 0
+						switch(rand(1, 11))
+							if (1) 
+								message = "<B>[mob]</B> sniffs."
+								playsound(get_turf(mob), "sound/voice/animal/gabe1.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (2) 
+								message = "<B>[mob]</B> barks?"
+								playsound(get_turf(mob), "sound/voice/animal/gabe2.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (3) 
+								message = "<B>[mob]</B> borks."
+								playsound(get_turf(mob), "sound/voice/animal/gabe3.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (4) 
+								message = "<B>[mob]</B> brorks."
+								playsound(get_turf(mob), "sound/voice/animal/gabe4.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (5) 
+								message = "<B>[mob]</B> hecks."
+								playsound(get_turf(mob), "sound/voice/animal/gabe5.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (6) 
+								message = "<B>[mob]</B> borfs."
+								playsound(get_turf(mob), "sound/voice/animal/gabe6.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (7) 
+								message = "<B>[mob]</B> bars?"
+								playsound(get_turf(mob), "sound/voice/animal/gabe7.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (8) 
+								message = "<B>[mob]</B> bjorks."
+								playsound(get_turf(mob), "sound/voice/animal/gabe8.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (9) 
+								message = "<B>[mob]</B> barks."
+								playsound(get_turf(mob), "sound/voice/animal/gabe9.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (10) 
+								message = "<B>[mob]</B> boos."
+								playsound(get_turf(mob), "sound/voice/animal/gabe10.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+							if (11) 
+								message = "<B>[mob]</B> brehs."
+								playsound(get_turf(mob), "sound/voice/animal/gabe11.ogg", 80, 0, 0, max(0.7, min(1.2, 1.0 + (30 - mob.bioHolder.age)/60)))
+						SPAWN_DBG(3 SECONDS)
```

Comment:
Is this consistent with how other custom ones are done? It would probably make more sense to use a `last_emote_time` then compare to the current time.

Reply:
The custom emote framework is largely copied from the custom emotes in the werewolf and zombie mutantrace entries, and is consistent as far as I know.

It also doesn't feel like it works properly, with repeated attempts to scream with alt-S causing the player to move downwards, and also throws an "unknown emote" message if I try to *howl after screaming (and vice-versa). I'll try to figure something out!